src/NativeShell/www/nativeshell.js

```diff
@@ -0,0 +1,44 @@
+
+
+function invokeMethod(successCallback, errorCallback, method, options) {
+    successCallback = successCallback || function () {};
+    errorCallback = errorCallback || function () {};
+    options = options || [];
+
+    cordova.exec(successCallback, errorCallback, 'NativeShell', method, options);
+}
+
+function NativeShell() {
+    this.FileSystem = require('org.jellyfin.mobile.FileSystem');
+    this.AppHost = require('org.jellyfin.mobile.AppHost');
```

Comment:
I am pretty sure the clobber section in `plugin.xml` will handle adding these two items.

Reply:
I will try defining the clobber as you suggest for the sub modules